packages/terra-form/src/Field.jsx

```diff
@@ -40,24 +52,56 @@ const propTypes = {
    * Determines whether the field is required.
    */
   required: PropTypes.bool,
+  /**
+   * Determines whether or not to append the 'optional' label to a non-required field label.
+   */
+  showOptional: PropTypes.bool,
 };
 
 const defaultProps = {
   children: null,
   error: null,
+  errorIcon: <IconError />,
```

Comment:
do we want to give them the option to pass in their own error icon?  Shouldn't we just lock this down like we have done with the error and no data views.

Reply:
I think for themability, we could be limited if we lockdown this icon.

Reply:
Given there are multiple sets of icons available with their own version of an Error Icon, this is needed for themability. It was specified in the implementation wireframe.